### ISSUE TITLE
Optional encoding kwarg for for `partition_{html, email, text}`

### ISSUE THREAD

**Issue by MthwRobinson** [Unstructured-IO/unstructured#214]
The goal of this issue is to add an optional `encoding` kwarg that allows users to pass in the encoding to bricks the process plain text files (`partiton_html`, `partition_email`, and `partition_text`). Currently, if you want to use a specific encoding, you need to read in the file yourself and pass it in as text, as shown below.

```python
with open("example-docs/example-10k.html", "r", encoding="utf-8") as f: #also passed encoding here
    text = f.read()
elements = partition_html(text=text)
```

**Comment by peter65374** [comment]
exactly, got same problem. encoding parameters should be pass to the wrapper.

**Comment by tomaarsen** [comment]
I believe this has been implemented successfully in #339, which is included in the latest `0.5.3` release. I would consider upgrading your `unstructured` to take advantage of this new functionality.
This issue can be closed, I think.

**Comment by MthwRobinson** [comment]
Yep, this one should be closed now.
